CMakeLists.txt

```diff
@@ -142,9 +142,12 @@ if(GFX_ENABLE_SCENE)
     target_include_directories(gfx PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/third_party/Vulkan-Headers/include)
     target_include_directories(gfx PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/third_party/tinyexr)
     target_link_libraries(gfx PUBLIC glm::glm)
-    target_link_libraries(gfx PRIVATE tinyobjloader ktx)
+    target_link_libraries(gfx PRIVATE tinyobjloader ktx tinyexr)
```

Comment:
tinyexr is now linked to the lib instead of pulling in the implementation from the ehader